imports/startup/client/index.ts

```diff
@@ -27,24 +25,13 @@ declare module 'vue' {
 	}
 }
 
-declare module '@vue/runtime-core' {
-	// Declare your own store states.
-	interface State {
-		auth: AuthState,
-		temporal: TemporalState,
-		navigation: NavigationState,
-	}
-
-	interface ComponentCustomProperties {
-		$store: Store<State>;
-	}
-}
-
 Meteor.startup(() => {
 	loadFonts();
 	const emitter = mitt();
+	const pinia = createPinia();
+	// @ts-ignore
```

Comment:
is this needed?

Reply:
yes, it was throwing mistype error with the line 33. It was an existing line